example/huobi/config.yaml

```diff
@@ -1,14 +1,15 @@
-defaultCashAccount: Assets:Huobi:Cash
-defaultPositionAccount: Assets:Huobi:Positions
-defaultCommissionAccount: Expenses:Huobi:Commission
-defaultPnlAccount: Income:Huobi:PnL
```

Comment:
Huobi's default accounts should not be `FIXME`, these default accounts are actually used in the transactions.